resources/azure/environment.go

```diff
@@ -64,6 +64,10 @@ func (e *Environment) InternalRegistry() string {
 	return "none"
 }
 
+func (e *Environment) InternalDockerhubMirror() string {
+	return "none"
```

Comment:
You can return public DockerHub so that it does not fail: `registry-1.docker.io`